### ISSUE TITLE
Issues installing requirements via webui-user.bat

### ISSUE THREAD

**Issue by hristo-naydenov** [CompVis/stable-diffusion#550]
I'm tring to install Stable Diffusion, but I'm running into some versioning issues. First I installed the latest available version of Python, which right now is 3.11.1, then I installed the latest version of pip, which right now is 22.3.1, and when I ran webui-user.bat, at some point it throws the following error:
`ERROR: Could not find a version that satisfies the requirement torch==1.12.1+cu113 (from versions: none)
ERROR: No matching distribution found for torch==1.12.1+cu113`

So then I look into this and reach the conclusion that pytorch doesn't seem to support any python version after 3.9. To quote their [documentation](https://pytorch.org/get-started/locally/)
`Currently, PyTorch on Windows only supports Python 3.7-3.9; Python 2.x is not supported.`

So then I downgrade Python to 3.8 and try to run the installation again, but then it throws this error:
`ERROR: Ignored the following versions that require a different python version: 2022 Requires-Python >=3.8,<4.0
ERROR: Could not find a version that satisfies the requirement blendmodes==2022 (from versions: 2020, 2020.1, 2020.2, 2020.2.1, 2020.2.2, 2020.3.0, 2021, 2021.1, 2021.2, 2021.3, 2021.3.1, 2021.3.2, 2021.3.3)
ERROR: No matching distribution found for blendmodes==2022`

So it appears that blendmodes 2022 is not supported any Pythonversions prior to 3.8. I'm a bit at a loss. What do I do?

**Comment by dgarrett01** [comment]
Same problem here. Seem to be stuck. Either you can't install torch, or blendmodes==2022 isn't supported.

**Comment by ed5464215** [comment]
I had the same issue. You need to 
1.Remove SD
2.Deinstall Python 3.11
3.Go to Control Panel and remove the old Win10 Python (in my case it was 3.7).
4.Open command prompt, type "python" and install Python 3.10 from Microsoft.
5.Install SD

**Comment by hristo-naydenov** [comment]
> I had the same issue. You need to 1.Remove SD 2.Deinstall Python 3.11 3.Go to Control Panel and remove the old Win10 Python (in my case it was 3.7). 4.Open command prompt, type "python" and install Python 3.10 from Microsoft. 5.Install SD

Installing python from the Microsoft Store didn't work for me as I couldn't start the runtime from CMD for some reason, even after adding the path to Python in my PATH env variable. So I uninstalled it, then I downloaded the 64-bit executable from https://www.python.org/downloads/release/python-3100/ and ran the SD installation again and it worked like a charm!

Thanks a lot for your help!
